pkg/util/mount/nsenter_mount.go

```diff
@@ -189,9 +189,10 @@ func (n *NsenterMounter) IsLikelyNotMountPoint(file string) (bool, error) {
 		return true, nil
 	}
 	strOut := strings.TrimSuffix(string(out), "\n")
-
 	glog.V(5).Infof("IsLikelyNotMountPoint findmnt output for path %s: %v", file, strOut)
-	if strOut == file {
+
+	//TODO: This is a temporary workaround for fixing the buggy code. Track the solution with issue #26996
+	if strings.Contains(file, strOut) {
```

Comment:
I'm not sure this is going to fix things.

What happens if `file` is `/a/likely/mountpoint-111` and `strOut` is `/a/likely/mountpoint-11`?